x-pack/plugins/uptime/public/lib/index.ts

```diff
@@ -6,4 +6,3 @@
  */
 
 export { MountWithReduxProvider } from './helper';
```

Comment:
this helper is also more or less related to test

Reply:
changed to the full path